src/main/java/scrumptious/TaskRunner.java

```diff
@@ -0,0 +1,102 @@
+package scrumptious;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Scanner;
+
+public class TaskRunner {
+    public static ArrayList<Task> tasks = new ArrayList<>();
+
+    public static void main(String[] args) {
```

Comment:
I think here the main implementation is for parser to be used for your testing?

Reply:
Yeap, Haoxiang, remove this file then commit again to the same branch, then we will merge this pull request

Reply:
ok I removed the file.
Is the scrumptious package name ok though

Reply:
I think it is ok, we can leave it as it is now and we will settle the package names later on

Reply:
ok. I merge? Or someone merge after checking or sth?

Reply:
I think wait for the next person to review then merge.